src/Components/getList.js

```diff
@@ -60,9 +64,8 @@ const FullList = (props) => {
         </li>
 
         {searchTerm
-           ? searchList.map(item => <GroceryItem item={item} />)
-           : props.data.map(item => <GroceryItem item={item} />)
-        }
+          ? searchList.map(item => <GroceryItem item={item} />)
```

Comment:
I think it would be valuable to have here `key={item.id}`, or whatever value is unique per item. This would help React know when to re-render specific items in the list.